docs/reference-guide/cookieplone-make-commands.md

```diff
@@ -0,0 +1,157 @@
+---
+myst:
+  html_meta:
+    "description": "Cookieplone make commands"
+    "property=og:description": "Cookieplone make commands"
+    "property=og:title": "Cookieplone make commands"
+    "keywords": "Cookieplone, make, commands"
+---
+
+# Cookieplone `make` commands
+
+This reference guide describes the function and purpose of all the `make` commands found in a {term}`Cookieplone` project.
+It's organized according to its components.
+-   Frontend
+-   Backend
+-   Documentation
+-   Environment
+-   Quality assurance (QA)
+-   Internationalization (i18n)
+-   Testing
+-   Container images
+-   Local stack
+-   Acceptance
+
+## Frontend
+
+When you issue a `make` command at the root of your project, you call the file {file}`Makefile` also at the root.
+In turn, it invokes commands in the file {file}`frontend/Makefile`.
+You can refer to these files for implementation details.
+You can run the following make targets by using the command structure of `make <TARGET>`.
+
+`frontend-install`
+:   Invokes the target `install` in `frontend/Makefile`.
+    Uses pnpm to run the mrs-developer tool with the default settings.
+    It then installs dependencies using pnpm.
+    After that, it runs `build-deps` to build dependency packages.  
```

Comment:
Not quite. See https://pnpm.io/cli/dlx and https://github.com/collective/mrs-developer. I had to look them up myself.

```make
	pnpm dlx mrs-developer missdev --no-config --fetch-https
	pnpm i
	make build-deps
```

```suggestion
    Uses `pnpm dlx` to run [mrs-developer](https://github.com/collective/mrs-developer) to retrieve dependencies in the file {file}`frontend/mrs.developer.json`.
    It then installs the dependencies.
    Finally, it builds the dependency packages `@plone/registry` and `@plone/components` from source.
```